src/app/shared/components/config-form-group/config-form-group.component.ts

```diff
@@ -57,6 +57,32 @@ export class ConfigFormGroupComponent implements OnInit, OnDestroy {
     private sanitizer: DomSanitizer,
   ) { }
 
+  public static listenToAllControlsOnDependencyChange(configFormGroup: ConfigFormGroup, toUnsubscribe: Array<Subscription>) {
+    [
+      ...configFormGroup.controlsRecursively,
+      configFormGroup
+    ]
+      .filter(c => c instanceof ConfigFormGroup || c instanceof ConfigFormControl)
+      .forEach((c: ConfigFormGroup | ConfigFormControl) => {
+        this.listenToOnDependencysChange(c, toUnsubscribe);
```

Comment:
Shouldn't be a static call from `ConfigFormGroupComponent` ?

Reply:
In this case it is permited because both methods are static

But you are right, the most correst one is to do static call, I'll fix it

Reply:
fixed